vast/vast.cpp

```diff
@@ -50,18 +50,31 @@ int main(int argc, char** argv) {
   }
   // Load plugins.
   auto loaded_plugin_paths = std::vector<std::filesystem::path>{};
-  auto plugin_files
+  auto plugin_files_or_names
```

Comment:
Slightly more exact:
```suggestion
  auto plugin_paths_or_names
```

Reply:
Good call. Will rename tomorrow.